src/ProxyClient/Http/InvalidationRequest.php

```diff
@@ -21,7 +21,7 @@
 class InvalidationRequest implements RequestInterface
 {
     use RequestDecorator;
```

Comment:
we don't require php-http/message-decorator explicitly, so tests fail because our request class does not respect the interface. probably some other lib previously required the decorator or something.